distributed/core.py

```diff
@@ -308,7 +308,11 @@ async def listen(self, port_or_addr=None, **kwargs):
             addr = port_or_addr
             assert isinstance(addr, str)
         listener = await listen(
-            addr, self.handle_comm, deserialize=self.deserialize, **kwargs,
+            addr,
+            self.handle_comm,
+            deserialize=self.deserialize,
+            allow_offload=False,
```

Comment:
Maybe this should also be parametrized to only be `False` on the scheduler? Since workers also use listeners if I'm not mistaken.

Reply:
Good point

Reply:
fixed